myRental/test/test.js

```diff
@@ -6,12 +6,17 @@ let bcrypt = require('bcryptjs');
 let exec = require('child_process').exec;
 let crypto = require('crypto');
 let faker = require('faker');
+let Log = require('../modules/Log.js');
 
 let DataBase = require('../modules/database.js');
-let User = require('../models/User.js').UserTest;
+let User = require('../models/UsersRequests.js').UserTest;
+let Request = require('../models/UsersRequests.js').RequestTest;
+let config = require('../config.json');
 
 let testUserRequest = generateFakeUserRequest();
+let testServiceRequest0 = generateFakeServiceRequest(0);
```

Comment:
This was confusing to look at. It initially looks like it makes a fake service request from some information stored in some array at index 0. However, I looked at the function, and the index isn't used, so you don't need to pass `0` in, and the name of this variable can be `testServiceRequest` or something that makes more sense.

Reply:
I forgot to change that back.